README.md

```diff
@@ -163,9 +165,10 @@ Platforms without FIPS compliance are also included in this jar, for compatibili
 
 Since version v0.29.20, GraalVM native image was supported. You can compile your application with AWS CRT in a GraalVM native image project without any additional configuration.
 
-Since version v0.30.12, GraalVM support was updated. Instead of packaging the JNI shared lib with native image as resource, the corresponding shared lib will be written to the same directory as the native image.
+Since version v0.31.1, GraalVM support was updated. Instead of packaging the JNI shared lib with native image as resource, the corresponding shared lib will be written to the same directory as the native image.
```

Comment:
Trivial: "shared lib with native image as resource," -> "shared lib with the native image as a resource," ?